src/fsharp/fscopts.fs

```diff
@@ -444,7 +444,6 @@ let testFlag tcConfigB =
 // not shown in fsc.exe help, no warning on use, motiviation is for use from VS
 let vsSpecificFlags (tcConfigB: TcConfigBuilder) = 
   [ CompilerOption("vserrors", tagNone, OptionUnit (fun () -> tcConfigB.errorStyle <- ErrorStyle.VSErrors), None, None);
-    CompilerOption("validate-type-providers", tagNone, OptionUnit (fun () -> tcConfigB.validateTypeProviders <- true), None, None);
```

Comment:
@dsyme yes I thought about that a bit, but dismissed it as it would need to be some kind of unsupported IDE setup... but I agree it's cheap and simple to keep it with a no-op so that we never have to worry.  Anyone is welcome to send that in a PR.